web/wp-content/themes/dctx-v2/assets/bower_components/jquery/src/var/support.js

```diff
@@ -0,0 +1,6 @@
+define( function() {
+	"use strict";
+
+	// All support tests are defined in their respective modules.
+	return {};
+} );
```

Comment:
:no_entry_sign: **Error**: Closing parenthesis of a multi-line function call must be on a line by itself (*PEAR.Functions.FunctionCallSignature.CloseBracketLine*).